packages/commons-server/src/libs/openapi-converter.ts

```diff
@@ -297,29 +301,46 @@ export class OpenAPIConverter {
               if (contentTypeHeader) {
                 if (version === 'SWAGGER') {
                   schema = routeResponse.schema;
+                  examples = routeResponse.examples;
                 } else if (version === 'OPENAPI_V3') {
                   schema = routeResponse.content?.[contentTypeHeader].schema;
+                  examples =
+                    routeResponse.content?.[contentTypeHeader].examples;
                 }
               }
 
-              routeResponses.push({
-                ...BuildRouteResponse(),
-                body: schema
-                  ? this.convertJSONSchemaPrimitives(
-                      JSON.stringify(
-                        this.generateSchema(schema),
-                        null,
-                        INDENT_SIZE
-                      )
-                    )
-                  : '',
-                statusCode: responseStatus === 'default' ? 200 : statusCode,
-                label: routeResponse.description || '',
-                headers: this.buildResponseHeaders(
-                  contentTypeHeaders,
-                  routeResponse.headers
-                )
-              });
+              const headers = this.buildResponseHeaders(
+                contentTypeHeaders,
+                routeResponse.headers
+              );
+
+              // add response based on schema
+              if (schema) {
```

Comment:
I think you got this check wrong (the tests in the desktop package broke, initially all the tests where in the desktop app as we don't have any lib). 
The idea was the following: 
Create a mockoon response when there is a status in the OpenAPI spec. Generate a sample body from the schema if any, and take all the other available params: description-> label, status-> statusCode. The response was always created and the schema was subject of a ternary to set the body or not.

As you wrapped everything in an `if`, this response is not generated anymore, and only the default one (line 348) gets generated. But this is supposed to happen only if there is no response at all in the OpenApi spec file.

Reply:
Thanks for pointing that out and for the explanation ! I fixed it.